.travis.yml

```diff
@@ -25,9 +25,9 @@ addons:
 env:
   global:
     - RAILS_ENV=test
-    - DATABASE_URL=postgres://postgres@localhost:5432/Forem_prod_test
+    - DATABASE_URL=postgres://travis@localhost:5432/Forem_prod_test
```

Comment:
username doesn't matter, because you changed _all_ localhost connections to "trusted" in the pg_hba.conf sed step, before we restarted